### PR TITLE
added comments feature

### PR DESCRIPTION
**Pull request by HelioStrike**
[Previous PR](https://github.com/HelioStrike/sps/pull/4)
[Website link](https://smutyala-sps-summer20.el.r.appspot.com)

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/Constants.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  public static final String TIMESTAMP = "timestamp";
+
+  public static final int COMMENT_MIN_LENGTH = 4;
+
+  public static final int COMMENT_MAX_LENGTH = 250;
+
+  public static final String AUTHOR_EMPTY_ERROR = "Author name cannot be empty.";
+
+  public static final String COMMENT_LENGTH_ERROR = "Comment should be between " + Integer.toString(COMMENT_MIN_LENGTH)
```
Comment (skashish-sps) [inline portfolio/src/main/java/com/google/sps/Constants.java]: You can leave this string in the file, as it is an error message.
Prefer using String Builder. It helps reduce memory usage

Comment (skashish-sps) [reply portfolio/src/main/java/com/google/sps/Constants.java]: Strings in JAVA are immutable, hence every '+' sign results in a new string object. String Builder uses a single object.

**Review by skashish-sps — COMMENTED** [review]
Just one comment, otherwise LGTM

**Comment by HelioStrike** [comment]
@skashish-sps Updated :)

**Review by skashish-sps — APPROVED** [review]
LGTM

**Pull request merged**